src/pages/whats-new.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "What's new in the MetaMask documentation"
+description: Summary of the latest updates to this documentation.
+---
+
+# What's new in the documentation
+
+The latest MetaMask documentation updates are listed by the month they were implemented. For a comprehenesive list of recent product changes, visit the [MetaMask developer page](https://metamask.io/developer/) or see the [release notes](https://github.com/MetaMask/metamask-sdk/releases).
+
+## February 2024
+- Added information about [ERC-4337](https://metamask.io/news/latest/account-abstraction-past-present-future/) - a unified standard that makes [account abstraction easier](/snaps/features/custom-evm-accounts/security). [(#1167)](https://github.com/MetaMask/metamask-docs/pull/1167)
+- Documented using [Snaps environment variables](/snaps/how-to/use-environment-variables). [(#1149)](https://github.com/MetaMask/metamask-docs/pull/1149)
+- Documented [Snaps static file API](/snaps/features/static-files). [(#1138)](https://github.com/MetaMask/metamask-docs/pull/1138)
+- Documented Snaps [unencrypted state](/snaps/reference/snaps-api/#snap_managestate). [(#1124)](https://github.com/MetaMask/metamask-docs/pull/1124)
+- Restructured [Snaps documentation](/snaps) for improved readability. [(#1129)](https://github.com/MetaMask/metamask-docs/pull/1129)
+- Added new [Snaps architecture diagrams](/snaps/learn/about-snaps). [(#1123)](https://github.com/MetaMask/metamask-docs/pull/1123)
+
+
+## January 2024
```

Comment:
Please be attentive to your spacing for source code readability. Always add line breaks around headers and don't use multiple line breaks anywhere.

Also, remember the [column limit](https://docs-template.consensys.io/contribute/format-markdown#column-limit).